### PR TITLE
Kubernetes Documentation edits

### PR DESCRIPTION
**Pull request by ruthryi**
Fix for https://github.com/wso2/docs-apim/issues/2077

### CODE REVIEW

**Comment by Mariangela** [comment]
@vithu30 , please review this PR.

File: en/docs/install-and-setup/install/deploying-api-manager-with-kubernetes-or-openshift-resources.md [hunk 1]
```diff
@@ -35,16 +35,16 @@ Follow the instructions below to use Kubernetes (K8s) and Helm resources for con
...
 
 2.  Provide the necessary configurations.
     
 
     !!! note
-        The default product configurations for deployment of WSO2 API Manager with WSO2 API Manager Analytics are available at `<HELM_HOME>/am-pattern-1/confs` folder. Change the configurations, as necessary.
+        The default product configurations for deployment of WSO2 API Manager with WSO2 API Manager Analytics are available [here](https://github.com/wso2/kubernetes-apim/tree/v3.2.0.2/advanced/am-pattern-1/templates) folder. Change the configurations, as necessary.
 
-    Open the `<HELM_HOME>/am-pattern-1/values.yaml` and provide the following values for WSO2 Subscription Configurations.
+    Open the `wso2am-pattern-1-secrets.yaml` and provide the following values for WSO2 Subscription Configurations.
```
Comment (vithu30) [inline en/docs/install-and-setup/install/deploying-api-manager-with-kubernetes-or-openshift-resources.md:47]: We need revert this change as suggested in - https://github.com/wso2/docs-apim/pull/2123 :)

**Review by vithu30 — APPROVED** [review]
